controllers/nnf_workflow_controller.go

```diff
@@ -595,6 +595,27 @@ func (r *NnfWorkflowReconciler) startSetupState(ctx context.Context, workflow *d
 			err := fmt.Errorf("Servers resource does not meet storage requirements for directive '%s'", dbd.Spec.Directive)
 			return nil, nnfv1alpha1.NewWorkflowError("Allocation request does not meet directive requirements").WithFatal().WithError(err)
 		}
+
+		for _, breakdownAllocationSet := range dbd.Status.Storage.AllocationSets {
+			found := false
+			for _, serverAllocationsSet := range s.Spec.AllocationSets {
+				if breakdownAllocationSet.Label != serverAllocationsSet.Label {
+					continue
+				}
+
+				found = true
+
+				if serverAllocationsSet.AllocationSize < breakdownAllocationSet.MinimumCapacity {
```

Comment:
This is a little weird to me, that found is ahead of an error condition that returns. IDK, personal preference.

Reply:
It's fine.

Sure wish Go had Python's for/else